### ISSUE TITLE
When switching to a podcast screen all icons are briefly shown

### ISSUE THREAD

**Issue by mchowning** [Automattic/pocket-casts-android#1187]
### Description

When loading switching to an already opened podcast screen, it briefly flashes that all the podcasts are videos, starred, and queued. This does not occur when initially loading the screen. 

### Step-by-step reproduction instructions

1. From the podcasts tab, open a podcast
2. Switch to another tab
3. Return to the podcasts tab
4. Observe that all the episodes briefly flash the icons for video, being queued, and starred

### Screenshots or screen recording

| Briefly flashed when returning to the screen | After screen is fully loaded |
|--------|--------|
| ![Screenshot 2023-07-20 at 2 34 24 PM](https://github.com/Automattic/pocket-casts-android/assets/4656348/75847f6d-8860-40c7-ae22-52443170c9f5) | ![Screenshot 2023-07-20 at 2 34 32 PM](https://github.com/Automattic/pocket-casts-android/assets/4656348/be60c7c2-ee07-44ee-8d8b-1a644cf6c7c7) |

https://github.com/Automattic/pocket-casts-android/assets/4656348/53ed815e-195c-4ade-b291-40efde4332ea

### Did you search for existing bug reports?

- [X] I have searched for existing bug reports.

### Device, Operating system, and Pocket Casts app version

Samsung A13 and Pixel 6 Pro
Android 13
7.43-rc-3

**Comment by neeldoshii** [comment]
Can i work on this issue?

**Comment by mchowning** [comment]
Absolutely. Thanks @neeldoshii !

**Comment by neeldoshii** [comment]
I don't see the icons are anytime there can you please elaborating how it is supposed to be. 

I am always seeing this no icons. @mchowning 

![image](https://github.com/Automattic/pocket-casts-android/assets/60827173/b4742700-65cb-4179-9596-e73fe85ccf68)

**Comment by mchowning** [comment]
I'm seeing the icons just briefly flashing when I first switch to the screen from another tab like in the video included in the issue description. So from the screenshot you have in your comment. Tap on the filters tab, and then tap on the Podcasts tab. I think that when the Podcasts tab first reloads you'll see the icons all briefly flash.

I'm guessing this is probably caused by some default state that gets shown before the ViewModel loads the actual data for teh different episodes. Just having the default state be to show no icons, would probably address this. An alternative would be to not show the row for an episode at all until all the data for that episode is loaded.

**Comment by neeldoshii** [comment]
Yes! Correct I got the issue. Do you want me to have the icons on it too or we need to remove it in the fix?

**Comment by neeldoshii** [comment]
1. I mean that do we need to have the icons on the Podcast page when its loaded 

> This means that as shown in your screenshot  ****##After screen is fully loaded##**** we will add the icons over there so it will maintain the consistency and from user point of view disturbance of visuals wont be there.

or 

2. The other scenario where we just remove the icons so it wont be shown at start neither after load. 

> This means that as shown in your screenshot  ***Briefly flashed when returning to the screen*** we just remove the icon from here

Thank you let me know what works best.

Also does Automattic have slack or some community place to interact with the internal team or with other contributors for networking and issue solving.

**Comment by neeldoshii** [comment]
> @neeldoshii their is a slack inquire with support & they can add you 😉

Can you provide me the link?

**Comment by CookieyedCodes** [comment]
Non staff can't generate a link sorry 😅
